packages/docs/src/components/DocsExample.vue

```diff
@@ -125,3 +112,27 @@ export default {
   }
 }
 </style>
+
+<style lang="scss" scoped>
+  .docs-example-card {
```

Comment:
Hmmm, why lint didn't work here? Should be no tab.

Reply:
Should be two spaces actually, like it is :)